docs/content/getting-started/database-component.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Database
```

Comment:
`wick new` is available now so at the end you should plug that as a good way to get started.

Reply:
Mentioned the `wick new` commands at the end of the tutorials